cmake/FindH5Part.cmake

```diff
@@ -1,32 +1,48 @@
-#------------------------------------------------------------------------------#
-# Copyright (c) 2016 Los Alamos National Security, LLC
-# All rights reserved.
-#------------------------------------------------------------------------------#
+################################################################################
+#
+# \file      cmake/FindH5Part.cmake
+# \author    C. Junghans
+# \copyright 2016, Los Alamos National Security, LLC.
+# \brief     Find the H5Part library
+# \date      Mon 05 Dec 2016 09:02:53 AM MST
+#
+################################################################################
```

Comment:
The whole header? Doxygen uses it to generate metadata on a file, e.g., http://quinoacomputing.github.io/quinoa/_lin_sys_merger_8h.html#details. Though you are right:

- this does not happen for the Find*.cmake files, and
- `git blame` is another way to do it.